1.9/api/access.md

```diff
@@ -0,0 +1,40 @@
+---
+post_title: Cluster Access
+nav_title: Cluster Access
+menu_order: 1
+---
+
+The cluster address may be obtained using the following methods:
+
+- Log into the DC/OS GUI and copy the scheme and domain name from the browser address bar.
+- Log into the DC/OS CLI and type `dcos config show core.dcos_url` to get the cluster URL.
+
+
+## API ports
+
+On the master nodes, Admin Router is accessible through standard ports: `80` (http) and `443` (https, if enabled).
+
+On the agent nodes, Admin Router Agent is accessible through port `61001` (http).
+
+
+## Agent node address
+
+The IP of specific agent nodes may be obtained using the following methods:
+
+- Log into the DC/OS GUI, navigate to the Nodes page, and copy the hostname of the desired node.
+- Log into the DC/OS CLI, list the nodes with `dcos node`, and copy the hostname of the desired node.
+
+To determine which agents are public agents, see [Finding a Public Agent IP](/docs/1.9/administration/locate-public-agent/).
+
+
+## Ingress
+
+In most production deployments, administrative access to the cluster should be routed through an external proxy to the DC/OS master nodes, distributing traffic load between the master nodes. For example, the default AWS templates configure an AWS Elastic Load Balancer.
+
+Master nodes and private agent nodes are usually not publicly accessible. For security reasons, ingress to these nodes should be controlled by a router or firewall. To manage the cluster, administrators and operators should use a VPN server inside the firewall, on the same network(s) as the DC/OS nodes. This way you can securely access the nodes directly from your workstation.
```

Comment:
s/s `The VPN server ensures that you can securely access the nodes directly from your workstation.`